plugin/package.json

```diff
@@ -14,14 +14,17 @@
     "start:terrain": "vite -c vite-configs/location/modals/terrain/web.ts",
     "start:sidebar": "vite -c vite-configs/sidebar/web.ts",
     "start:location": "vite -c vite-configs/location/web.ts",
+    "start:geolocation": "vite -c vite-configs/geolocation/web.ts",
     "build": "run-s typecheck clean build:plugin",
-    "build:plugin": "run-s build:sidebar build:location",
+    "build:plugin": "run-s build:sidebar build:location build:geolocation",
     "build:sidebar": "run-s build:datacatalog-web; vite build -c vite-configs/sidebar/web.ts; vite build -c vite-configs/sidebar",
     "build:location": "run-s build:google-web build:terrain-web build:location-web; vite build -c vite-configs/location",
     "build:location-web": "vite build -c vite-configs/location/web.ts",
     "build:google-web": "vite build -c vite-configs/location/modals/googleAnalytics/web.ts",
     "build:terrain-web": "vite build -c vite-configs/location/modals/terrain/web.ts",
     "build:datacatalog-web": "vite build -c vite-configs/sidebar/modals/datacatalog/web.ts",
+    "build:geolocation": "run-s build:geolocation-web;vite build -c vite-configs/geolocation",
```

Comment:
Please move this up with the other plugin builds. I think its better to keep plugin and web build scripts separate

Reply:
fc8de4437577d111f90b0a1c7db09d37c48d4de6

Reply:
Sorry I think you misunderstood. I was hoping you could move the whole build:geolocation script up here, not add the web build to the build:plugin script.
Similar to keeping component props split between variables and functions, I think its cleaner to keep the builds grouped (plugin, then web)

Reply:
done